__tests__/components/Button.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { render, fireEvent } from '@testing-library/react'
+import Button, { noop } from '../../components/Button'
+
+describe('Button Component', () => {
+  test('noop func', () => {
```

Comment:
Had to test `noop` function for 100% coverage, we may want to look into putting this into a global helper function to be used and tested individually

Reply:
We might also rename it. (Maybe noOperationFunction, dummyFunc ...etc)
`noop` naming was not straightforward for me I had to google it.
Code should be easy to read.

Reply:
`noop` is a standard name for this type of function though

Reply:
Hoo cool I didn't know 🙂

Reply:
@coltonehrman is right, `noop` is standard name for useless function. Next time, the information comment should live with the code (add a comment in the codebase)